### DISCUSSION TITLE
Arbitrary values color with opacity tailwind

### DISCUSSION THREAD

**Discussion by VictorPulzz** [tailwindlabs/tailwindcss#9068] [Help]
Hello!
Сan you help me?
I want add dynamic color with opacity in component with css variables
Component:
```ts
 <div
      className={styles.block}
      style={{  '--color': color} as CSSProperties}
   />
```
Scss module: 
```css
.block {
  @apply text-[color:var(--color)] bg-[color:var(--color)]/10;
}
```
This class `bg-[color:var(--color)]/10` not work(

**Comment by wongjn** [comment]
You could set the `--color` to a color preset in Tailwind config as per the [documentation](https://tailwindcss.com/docs/customizing-colors#using-css-variables):

```js
module.exports = {
  theme: {
    extend: {
      colors: {
        dynamic: 'rgb(var(--color) / <alpha-value>)'
      },
    },
  },
};
```

Which would then allow you to use your `--color`:

```scss
.block {
  @apply text-dynamic bg-dynamic/10;
}
```

I don't think your current syntax could ever work, since Tailwind would not know at build time what format `--color` is, RGB, HSL, HEX, etc.
